infrastructure/main.tf

```diff
@@ -7,7 +7,38 @@ resource "azurerm_resource_group" "rg" {
 }
 
 locals {
-  ase_name = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  ase_name               = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  is_preview             = "${(var.env == "preview" || var.env == "spreview")}"
+  previewVaultName       = "${var.product}-bsp"
+  nonPreviewVaultName    = "${var.product}-bsp-${var.env}"
+  vaultName              = "${local.is_preview ? local.previewVaultName : local.nonPreviewVaultName}"
+  vaultResourceGroupName = "rpe-bulk-scan-processor-${var.env}"
+}
+
+module "bulk-scan-key-vault" {
+  source                  = "git@github.com:hmcts/moj-module-key-vault?ref=master"
+  name                    = "${local.vaultName}"
+  product                 = "${var.product}"
+  env                     = "${var.env}"
+  tenant_id               = "${var.tenant_id}"
+  object_id               = "${var.jenkins_AAD_objectId}"
+  resource_group_name     = "${local.vaultResourceGroupName}"
+  # dcd_cc_dev group object ID
+  product_group_object_id = "38f9dea6-e861-4a50-9e73-21e64f563537"
+}
+
+module "servicebus-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace.git"
+  name                = "${var.product}-${var.component}-servicebus-${var.env}"
```

Comment:
could we remove service bus from the name?
https://tools.hmcts.net/confluence/display/CNP/Naming+Conventions